go.mod

```diff
@@ -10,10 +10,11 @@ require (
 	github.com/mattn/go-runewidth v0.0.6 // indirect
 	github.com/mattn/go-sqlite3 v1.11.0
 	github.com/olekukonko/tablewriter v0.0.1
+	github.com/pingcap/parser v0.0.0-20190613045206-37cc370a20a4
+	github.com/pingcap/tidb v0.0.0-20190625145607-60965b006877
```

Comment:
Should we fix the version of TiDB as something like [`v3.0.5`](https://github.com/pingcap/tidb/tree/v3.0.5)?

Reply:
This line is generated by running `go get github.com/pingcap/tidb@v3.0.0`. I am not sure why it appears to be `v.0.0.0-2019....` in the `go.mod`.